root/report/RecordingsSameNameDifferentArtistsSameName.js

```diff
@@ -28,44 +26,30 @@ const RecordingsSameNameDifferentArtistsSameName = ({
   generated,
   items,
   pager,
-}: ReportDataT<ReportRecordingT>): React.Element<typeof Layout> => (
-  <Layout
+}: ReportDataT<ReportRecordingT>): React.Element<typeof ReportLayout> => (
+  <ReportLayout
     $c={$c}
-    fullWidth
-    title={l(`Recordings with the same name by different artists
-              with the same name`)}
+    canBeFiltered={canBeFiltered}
+    description={exp.l(
+      `This report shows all recordings with the same name that have
+       different artists (having different MBIDs) with the same name.
+       These are most likely cases where the {ac|artist credit} is
+       incorrect for at least one of the recordings.`,
+      {ac: '/doc/Artist_Credits'},
+    )}
+    entityType="recording"
+    extraInfo={l(
+      `Currently, this report only works
+       with recordings that have one artist.`,
+    )}
+    filtered={filtered}
+    generated={generated}
+    title={l(
+      `Recordings with the same name
+       by different artists with the same name`,
+    )}
+    totalEntries={pager.total_entries}
   >
-    <h1>
-      {l(`Recordings with the same name by different artists
-          with the same name`)}
-    </h1>
-
-    <ul>
-      <li>
-        {l(`This report shows all recordings with the same name that have
-            different artists (having different MBIDs) with the same name.`)}
-      </li>
-      <li>
-        {exp.l(`These are most likely cases where the {ac|artist credit} is
-                incorrect for at least one of the recordings.`,
-               {ac: '/doc/Artist_Credits'})}
-      </li>
```

Comment:
I joined these two into one string here, since this was the only case where we had *three* paragraphs (and it seemed silly to support three sections just for this). I see no reason to have these two separate anyway - most reports have "this is what this report is about" in one string.